library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java

```diff
@@ -291,53 +477,10 @@ private void editMessage(Message message) {
         }
     }
 
-    private void initAttachmentUI() {
-        // TODO: make the attachment UI into it's own view and allow you to change it.
-        messageInputController = new MessageInputController(getContext(), binding, this.viewModel, style);
-        binding.rvMedia.setLayoutManager(new GridLayoutManager(getContext(), 4, RecyclerView.VERTICAL, false));
-        binding.rvMedia.hasFixedSize();
-        binding.rvComposer.setLayoutManager(new GridLayoutManager(getContext(), 1, LinearLayoutManager.HORIZONTAL, false));
-        int spanCount = 4;  // 4 columns
-        int spacing = 2;    // 1 px
-        boolean includeEdge = false;
-        binding.rvMedia.addItemDecoration(new GridSpacingItemDecoration(spanCount, spacing, includeEdge));
-        binding.tvClose.setOnClickListener(v -> {
-            messageInputController.onClickCloseBackGroundView();
-            if (viewModel.isEditing()) {
-                initSendMessage();
-                clearFocus();
-            }
-        });
-
-        binding.llMedia.setOnClickListener(v -> messageInputController.onClickOpenSelectMediaView(v, null));
-
-        binding.llCamera.setOnClickListener(v -> {
-            if (!PermissionChecker.isGrantedCameraPermissions(getContext())) {
-                PermissionChecker.showPermissionSettingDialog(getContext(), getContext().getString(R.string.stream_camera_permission_message));
-                return;
-            }
-            Utils.setButtonDelayEnable(v);
-            messageInputController.onClickCloseBackGroundView();
-
-            Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
-            ContentValues values = new ContentValues();
-            values.put(MediaStore.Images.Media.TITLE, "New Picture");
-            values.put(MediaStore.Images.Media.DESCRIPTION, "From your Camera");
-            imageUri = getContext().getContentResolver().insert(
-                    MediaStore.Images.Media.EXTERNAL_CONTENT_URI, values);
-            takePictureIntent.putExtra(MediaStore.EXTRA_OUTPUT, imageUri);
-
-            Intent takeVideoIntent = new Intent(MediaStore.ACTION_VIDEO_CAPTURE);
-            Intent chooserIntent = Intent.createChooser(takePictureIntent, "Capture Image or Video");
-            chooserIntent.putExtra(Intent.EXTRA_INITIAL_INTENTS, new Intent[]{takeVideoIntent});
-
-            if (this.openCameraViewListener != null)
-                openCameraViewListener.openCameraView(chooserIntent, Constant.CAPTURE_IMAGE_REQUEST_CODE);
-        });
-        binding.llFile.setOnClickListener(v -> messageInputController.onClickOpenSelectFileView(v, null));
-    }
-
     // endregion
+
+    // region permission check
+    // TODO: the name of this method is weird (progres..)? perhaps captureMedia?
```

Comment:
Seems `TODO` might be removed.

Reply:
I have removed this in the other PR so it will be removed soon.